subiquity/ui/views/filesystem/lvm.py

```diff
@@ -205,6 +221,17 @@ def done(self, sender):
         del result["size"]
         mdc = self.form.devices.widget
         result["devices"] = mdc.active_devices
+        if "create_recovery_key" in result:
+            if result["create_recovery_key"]:
+                backup_prefix = pathlib.Path("/var/log/installer")
+                backup_filename = pathlib.Path(f"recovery-key-{result['name']}.txt")
+                result["recovery-key"] = {
+                    "backup-location": str(backup_prefix / backup_filename),
+                    "live-location": str(
+                        pathlib.Path("~").expanduser() / backup_filename
```

Comment:
the use of the term "backup" sounds like a copy-paste error but really it's fine, so maybe rename backup_filename to a term that users neither the word "backup" nor "live".

Reply:
Good point! Renaming to "filename".